bin/p2ptrans

```diff
@@ -129,7 +129,8 @@ def main():
         
     print("=>Producing the steps along the transition<=")
     result = produceTransition(n_steps, tmat, dispStruc, vec_classes,
-                               outdir, anim or savedisplay or interactive)
+                               outdir, anim or savedisplay or interactive,
+                               A.name, B.name)
```

Comment:
Can you explicitely specify `a_name=A.name` and `b_name=B.name`. You are skipping over the `habit` optional argument so I think this might produce an error.

Reply:
Does habit need to also need to be explicitly specified, or does it just work as a positional?

Reply:
No, it's optional. The problem is it was the first optional argument, so python would have thought that `A.name` is the input for habit